client-common/src/cipher/AttestationReportSigningCACert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is this certificate necessary?
can user make it?

Reply:
I don't think so, it hard coded in the ra-client.